src/components/periodic-report/dto/due-report.dto.ts

```diff
@@ -0,0 +1,17 @@
+import { Field, ObjectType } from '@nestjs/graphql';
+import { SecuredProperty } from '../../../common';
+import { IPeriodicReport, PeriodicReport } from './periodic-report.dto';
+
+@ObjectType()
+export class DueReport {
```

Comment:
Does it make more sense for this DTO & field resolvers to be plural since they are referring to multiple reports?


Also just FYI since this doesn't have an ID field we'll want it to be cached on the parent object in the UI
```ts
// api/typePolicies.base.ts
DueReport: { keyFields: false },
```